pkg/config/toml/config.go

```diff
@@ -669,6 +670,7 @@ type GasEstimator struct {
 	LimitTransfer   *uint64
 	LimitJobType    GasLimitJobType `toml:",omitempty"`
 	EstimateLimit   *bool
+	SenderAddress   *string
```

Comment:
You can use a `types.EIP55Address` here too:
```suggestion
	SenderAddress   *types.EIP55Address
```